src/client/utils/Sort.ts

```diff
@@ -523,6 +540,17 @@ export function getLabelsRaw(cube: Card[] | null, sort: string, showOther: boole
       }
     }
     ret = res;
+  } else if (sort === 'Word Count') {
+    const labels: string[] = [];
+    for (const card of cube || []) {
+      const wordCount = cardWordCount(card);
+      const label = wordCountBucket(wordCount);
+      if (!labels.includes(label)) {
+        labels.push(label);
+        if (labels.length === wordCountBuckets.length) break;
```

Comment:
small nit: for style consistency would be nice to have {} brackets with the if and break